### PR TITLE
refresh list of contact, group & user after add/edit

### PR DESCRIPTION
**Pull request by tenzap**
Fixes: https://github.com/kalkun-sms/Kalkun/issues/244

### CODE REVIEW

File: application/views/js_init/phonebook/js_phonebook.php [hunk 1]
```diff
@@ -45,11 +45,13 @@
 											buttons: {
 												"Okay": function() {
 													$(this).dialog("close");
+													location.reload();
```
Comment (kingster) [inline application/views/js_init/phonebook/js_phonebook.php:48]: A reload of page on button close isn't the best experience. We should instead trigger the ajax reload of messages if possible (don't recall how its implement in other cases) .

Comment (tenzap) [reply application/views/js_init/phonebook/js_phonebook.php:48]: Ok, I have no interest in learning ajax. Maybe you can do it now that you know where the issue lies then. Thanks.

Comment (tenzap) [reply application/views/js_init/phonebook/js_phonebook.php:48]: Just saw that when adding contacts by csv, the page is also reloaded in code with load->view...

Comment (kingster) [reply application/views/js_init/phonebook/js_phonebook.php:48]: Hmm, looks like all pages in kalkun is made without any aync/ajax calls and are rendered from PHP only.

I think   `location.reload();` here is not required. the one at line 54 is enough.

Comment (tenzap) [reply application/views/js_init/phonebook/js_phonebook.php:48]: I guess both are needed, one if the user clicks close, one if the user doesn't click but the dialog auto closes after timeout. Am I wrong?

**Comment by tenzap** [comment]
@kingster any news here?

**Comment by tenzap** [comment]
please check the new method

File: application/views/main/phonebook/contact/pbk_list.php [hunk 2]
```diff
@@ -1,4 +1,5 @@
-<?php //$this->load->view('js_init/phonebook/js_phonebook');
+<?php
+$this->load->view('js_init/phonebook/js_phonebook');
```
Comment (kingster) [inline application/views/main/phonebook/contact/pbk_list.php:2]: Let's move this JS block to end of this view/file.

**Comment by kingster** [comment]
@tenzap I think I found the root cause for the issue, It seems we need to change the bind strategy to resolve the root cause https://stackoverflow.com/questions/16598213/how-to-bind-events-on-ajax-loaded-content. This way we won't need the change in https://github.com/kalkun-sms/Kalkun/commit/6a49e881d900e04a6c0da1e0233090ace95f304b

**Comment by tenzap** [comment]
thanks, should be fine now

**Review by kingster — APPROVED** [review]

**Pull request merged**